rumqttc/CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ To update your code simply remove `Key::ECC()` or `Key::RSA()` from the initiali
 - Make v5 `RetainForwardRule` public, in order to allow setting it when constructing `Filter` values.
 - Use `VecDeque` instead of `IntoIter` to fix unintentional drop of pending requests on `EventLoop::clean` (#780)
 - `StateError::IncommingPacketTooLarge` is now `StateError::IncomingPacketTooLarge`.
+- Update `tokio-rustls` to `0.25.0`, `rustls-native-certs` to `0.7.0`, `rustls-webpki` to `0.102.1`,
+  `rusttls-pemfile` to `2.0.0`, `async-tungstenite` to `0.24.0`, `ws_stream_tungstenite` to `0.12.0`
+  and `http` to `1.0.0`. This is a breaking change as types from some of these crates are part of
```

Comment:
this isn't a breaking change right? as none of the functionality / API exposed to users is changing? :eyes:

Reply:
The `rustls` types are part of `rumqttc`'s public API so this is a breaking change. Clients which depend on `rustls` 0.21 and `rustls-native-certs` 0.6 and call `rumqttc` methods using their types will break if they update `rumqttc` without updating the other dependencies to match.